packages/cli/src/ldap.ee/ldap.service.ee.ts

```diff
@@ -154,7 +154,7 @@ export class LdapService {
 				config.set(LDAP_LOGIN_ENABLED, false);
 				await setCurrentAuthenticationMethod('email');
 			}
-		} else {
+		} else if (enabled) {
```

Comment:
Personal opinion, but I always find more readable early returns for validation statement.
Suggested refacto:
```ts
private async setLdapLoginEnabled(enabled: boolean): Promise<void> {
	const currentMethod = getCurrentAuthenticationMethod();

	if (enabled && !isEmailCurrentAuthenticationMethod() && !isLdapCurrentAuthenticationMethod()) {
		throw new InternalServerError(
			`Cannot enable LDAP login when current authentication method is '${currentMethod}'`
		);
	}

	config.set(LDAP_LOGIN_ENABLED, enabled);
	await setCurrentAuthenticationMethod(enabled ? 'ldap' : 'email');
}
```